woocommerce/woocommerce-gutenberg-products-block
# Update innerBlock template for Featured Product and Featured Category blocks

This PR updates the inner block template for Featured Product and Featured Category. This is necessary because a button block should have as the parent the buttons block. In this way, the user is able to customize again:
- font-size
- width settings

**NOTE:** The width is customizable only when the alignment is set to `none`. This is the default behavior on Gutenberg.



Fixes #5676





### Screenshots



### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Add a Featured Product or Featured Category block to a post or page.
2. Click on the inner button.
3. Try changing the Typography size.
4. Set alignment to none and change the width.
5. Check that the changes are visible on the editor side and frontend side.
### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).

### Changelog
> Featured Category and Featured Product: fix width and font size customization for the inner button block.
